src/algebra/order/floor.lean

```diff
@@ -214,6 +214,68 @@ ext $ λ x, ceil_eq_iff hn
 
 /-! #### Intervals -/
```

Comment:
If you `open set` (here or at the beginning of the file), you can write just `Icc` instead of `set.Icc`, and similarly for all your other intervals.

Reply:
Presumably I'd also need to modify much of the rest of the file to use this `open` (see the `preimage` lemmas below). Happy to do so, but want to confirm that's the goal before giving you a much bigger diff!